be/src/exec/vectorized/hdfs_scanner_parquet.cpp

```diff
@@ -29,6 +30,8 @@ void HdfsParquetScanner::do_update_counter(HdfsScanProfile* profile) {
 
     RuntimeProfile* root = profile->runtime_profile;
     ADD_COUNTER(root, kParquetProfileSectionPrefix, TUnit::UNIT);
+    parsed_bytes_read = ADD_CHILD_COUNTER(root, "ParsedBytesRead", TUnit::BYTES, kParquetProfileSectionPrefix);
```

Comment:
How to understand the `parsed` in this profile name? It seems represents the page bytes read.

Reply:
OK. fixed